shared/components/button/Button.jsx

```diff
@@ -4,8 +4,9 @@ import styled from 'styled-components';
 const Button = styled.button`
   background-color: ${props =>
     props.disabled === false ? '#39b54a' : '#f5f7f9'};
+  border: 1px solid #ced0da;
   border-radius: 4px;
-  border: ${props => (props.disabled === false ? '0' : '1px solid #ced0da')};
+  border-width: ${props => (props.disabled === false ? '0' : '1px')};
```

Comment:
Now you're actually introducing more code. It wasn't intended to have you change this, but what I meant was:
`border: ${props => !props.disabled ? 0 : '1px'} solid #ced0da;`

Reply:
Nice solution!